dali/operators/reader/loader/video_loader.cc

```diff
@@ -585,10 +590,16 @@ void VideoLoader::read_file() {
           if (!is_first_frame) {
             nonkey_frame_count = 0;
             if (frame > req.frame + req.count) {
```

Comment:
Shouldn't we check for:
```suggestion
            if (frame > req.frame + (1 + key_frames_count) * req.count) {
```
or something?
I mean, if key frame is every `req.count` (just guessing), we need to see the multiple of the distance to detect key frame.

Reply:
No, keyframe is the video stream property and it doesn't have to be equally placed in the video. `req.count` is how many frames we need to decode to between first and the last frame in the sequence.